config/dr-cluster/manager/ramen_manager_config.yaml

```diff
@@ -14,3 +14,5 @@ maxConcurrentReconciles: 50
 volSync:
   destinationCopyMethod: Direct
 ramenOpsNamespace: ramen-ops
+multiNamespace:
+  FeatureEnabled: true
```

Comment:
Why not "enabled" - more consisstent with kubeObjectProtection and volSync ("disabled")?

Reply:
Discussed in meeting, not new, and we cannot change it now.